tests/ui/const-generics/const-arg-in-const-arg.rs

```diff
@@ -2,8 +2,8 @@
 // we use a single revision because this should have a `full` revision
 // but right now that ICEs and I(@BoxyUwU) could not get stderr normalization to work
 
-#![cfg_attr(full, feature(generic_const_exprs))]
-#![cfg_attr(full, allow(incomplete_features))]
+// #![cfg_attr(full, feature(generic_const_exprs))]
```

Comment:
Probably remove these instead of commenting them out? Here and in another file.

Reply:
I'm fine either way, I can see the justification for leaving them in because they *should* work but they don't because of bugs and limitations.

Reply:
I prefer to let them since they are supposed to be used and will potentially to be used in future but aren't due to bugs and limitations.

In particular if you look at the top of this file you can see a lengthy explanation of why there is currently no "full" revision (spoiler: ICE messed up things).